packages/quantic/force-app/main/default/lwc/quanticCitation/__tests__/quanticCitation.test.js

```diff
@@ -242,8 +242,10 @@ describe('c-quantic-citation', () => {
       await citationLink.dispatchEvent(
         new CustomEvent('mouseenter', {bubbles: true})
       );
+      // The tooltip should not be shown immediately
+      expect(showTooltipSpy).toHaveBeenCalledTimes(0);
+      // Wait 200ms debounce duration and verify tooltip is shown
```

Comment:
```suggestion
      expect(showTooltipSpy).toHaveBeenCalledTimes(0);
```
Personally I don't think the comments are necessary to understand the code.